Add an openfermion interface (#272)
HQSquantumsimulations/struqture: documentation/src/openfermion.md

> # Interface with OpenFermion

Reviewer: ```suggestion
# Interface to OpenFermion
```

> Struqture also has an interface with the [OpenFermion](https://quantumai.google/openfermion) package allowing users to switch from one package to the other.

Reviewer: ```suggestion
Struqture also has an interface to the [OpenFermion](https://quantumai.google/openfermion) package allowing users to switch from one package to the other.
```

> For now only the interface for `PauliHamiltonian` is implemented with the functions `struqture_to_openfermion` and `openfermion_to_struqture`.

Reviewer: ```suggestion
For now only the conversion to and from `PauliHamiltonian` is implemented with the functions `struqture_to_openfermion` and `openfermion_to_struqture`.
```
